getGeocode.js

```diff
@@ -0,0 +1,66 @@
+const dotenv = require("dotenv");
+//note: this only works locally; in Lambda we use environment variables set manually
+dotenv.config();
+
+const fetch = require("node-fetch");
+const { Client } = require("@googlemaps/google-maps-services-js");
+const { generateKey } = require("./data/dataDefaulter");
+
+const getGeocode = async (name, street, zip) => {
+    const address = `${name},MA,${street},${zip}`;
```

Comment:
FYI I had to add the `MA` in order to get some of the geocoding results to get the right location when the `street` and `zip` are missing. If I didn't add this, some of the CVS locations resolved to a CVS in a different state.

Reply:
So I ended up adding the axios retry (Google allows retries in their JavaScript package using axios) and configured an exponential delay. Axios will attempt 5 retries before giving up.